assets/js/modules/adsense/settings/settings-main.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * AdSense Main Settings component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import SettingsEdit from './settings-edit';
+import SettingsView from './settings-view';
+
+export default function SettingsMain( { isOpen, isEditing } ) {
```

Comment:
I think this is the component that needs to handle the rollback, similar to: https://github.com/google/site-kit-wp/blob/develop/assets/js/modules/analytics/settings/settings-main.js#L34-L42